tests/test_pipelines.py

```diff
@@ -108,6 +108,17 @@ def test_download_only_pytorch(self):
             # We need to never convert this tiny model to safetensors for this test to pass
             assert not any(f.endswith(".safetensors") for f in files)
 
+    def test_force_safetensors_error(self):
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            # pipeline has Flax weights
+            with self.assertRaises(EnvironmentError):
+                tmpdirname = DiffusionPipeline.download(
+                    "hf-internal-testing/tiny-stable-diffusion-pipe",
```

Comment:
```suggestion
                    "hf-internal-testing/tiny-stable-diffusion-pipe-no-safetensors",
```

I just re-uploaded all the weights to a different repo just in case we do add safetensors to them :grin: 
https://huggingface.co/hf-internal-testing/tiny-stable-diffusion-pipe-no-safetensors